apps/backend/src/lambdas/reconcile.ts

```diff
@@ -19,9 +22,13 @@ export const handler = async (event: ReconciliationConfigInput) => {
   const cashExceptionsService = app.get(CashExceptionsService);
   const cashReconciliationService = app.get(CashReconciliationService);
   const posReconciliationService = app.get(PosReconciliationService);
+  const cashDepositService = app.get(CashDepositService);
+  const paymentService = app.get(PaymentService);
+  const posDepositService = app.get(PosDepositService);
   const locationService = app.get(LocationService);
   const reportingService = app.get(ReportingService);
   const appLogger = app.get(Logger);
+  const dateRange = { minDate: event.period.from, maxDate: event.period.to };
```

Comment:
Maybe we can add a check to say, "if it is todays date, then subtract one business day" to ensure we're not reconciling data on the current date?

Reply:
Will not do